readthedocs/projects/constants.py

```diff
@@ -383,43 +383,40 @@
     'https://bitbucket.org/{user}/{repo}/'
     'src/{version}{docroot}{path}{source_suffix}'
 )
-BITBUCKET_COMMIT_URL = (
-    'https://bitbucket.org/{user}/{repo}/'
-    'commits/{commit}'
-)
+BITBUCKET_COMMIT_URL = "https://bitbucket.org/{user}/{repo}/commits/{commit}"
 GITLAB_URL = (
-    'https://gitlab.com/{user}/{repo}/'
-    '{action}/{version}{docroot}{path}{source_suffix}'
-)
-GITLAB_COMMIT_URL = (
-    'https://gitlab.com/{user}/{repo}/'
-    'commit/{commit}'
+    "https://gitlab.com/{user}/{repo}/"
+    "{action}/{version}{docroot}{path}{source_suffix}"
 )
+GITLAB_COMMIT_URL = "https://gitlab.com/{user}/{repo}/commit/{commit}"
 GITLAB_MERGE_REQUEST_COMMIT_URL = (
-    'https://gitlab.com/{user}/{repo}/'
-    'commit/{commit}?merge_request_iid={number}'
-)
-GITLAB_MERGE_REQUEST_URL = (
-    'https://gitlab.com/{user}/{repo}/'
-    'merge_requests/{number}'
+    "https://gitlab.com/{user}/{repo}/commit/{commit}?merge_request_iid={number}"
 )
+GITLAB_MERGE_REQUEST_URL = "https://gitlab.com/{user}/{repo}/merge_requests/{number}"
 
 # Patterns to pull merge/pull request from providers
-GITHUB_PR_PULL_PATTERN = 'pull/{id}/head:external-{id}'
-GITLAB_MR_PULL_PATTERN = 'merge-requests/{id}/head:external-{id}'
+GITHUB_PR_PULL_PATTERN = "pull/{id}/head:external-{id}"
+GITLAB_MR_PULL_PATTERN = "merge-requests/{id}/head:external-{id}"
 
 # Git provider names
-GITHUB_BRAND = 'GitHub'
-GITLAB_BRAND = 'GitLab'
+GITHUB_BRAND = "GitHub"
+GITLAB_BRAND = "GitLab"
 
 # SSL statuses
-SSL_STATUS_VALID = 'valid'
-SSL_STATUS_INVALID = 'invalid'
-SSL_STATUS_PENDING = 'pending'
-SSL_STATUS_UNKNOWN = 'unknown'
+SSL_STATUS_VALID = "valid"
+SSL_STATUS_INVALID = "invalid"
+SSL_STATUS_PENDING = "pending"
+SSL_STATUS_UNKNOWN = "unknown"
 SSL_STATUS_CHOICES = (
-    (SSL_STATUS_VALID, _('Valid and active')),
-    (SSL_STATUS_INVALID, _('Invalid')),
-    (SSL_STATUS_PENDING, _('Pending')),
-    (SSL_STATUS_UNKNOWN, _('Unknown')),
+    (SSL_STATUS_VALID, _("Valid and active")),
+    (SSL_STATUS_INVALID, _("Invalid")),
+    (SSL_STATUS_PENDING, _("Pending")),
+    (SSL_STATUS_UNKNOWN, _("Unknown")),
+)
+
+SINGLE_VERSION = "single_version"
+MULTI_VERSION = "multi_version"
+VERSIONING_SCHEME_CHOICES = (
+    (SINGLE_VERSION, _("Single version")),
+    (MULTI_VERSION, _("Multi version")),
```

Comment:
I worry about this language when we introduce `Single language` projects. It seems like we need a better way to describe these projects that takes into account translations as well?

Reply:
So, first I named these:

- Single version, single language
- Multi version, multi language
- Multi version, single language (the new option)

But it felt too verbose, and I was hoping that having the preview of each case would clarify things better.

Maybe just re-think this as:

- Single version without translations
- Multiple versions with translations
- Multiple versions without translations (the new option)

Still verbose, but maybe more clear for users that aren't familiar with our language.

Reply:
I think explicit is probably best to start, if there isn't an obvious other way to describe it. I think the main thing is actually explaining the URL path (eg. `Multiple versions without translations (/<version>/<file>)`

Reply:
Do you mean use a placeholder instead of a live preview? Right now I'm showing how the URL will look like for the project (`/latest/file.html`, `/en/latest/file.html`, `/projects/en/latest`, using their language, subproject prefix, etc)

Reply:
That is great -- I think a live prefix is the most useful way to understand this. On top of that, I think we can just use the explicit naming above,.

Reply:
IMHO, `Multiple versions without translations (/<version>/<file>)` with a _static_ example is the best explanation. It may be verbose, but it removes all type of confusions.

Even if you don't know what's the meaning of `/en/` in the URL, if you want to host documentation in different languages you will select the option that says "... with translations"

Reply:
I'm fine with both options, static or dynamic preview.

Reply:
I think static is better 👍